src/components/Popover/index.native.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {propTypes, defaultProps} from './PopoverPropTypes';
+import CONST from '../../CONST';
+import Modal from '../Modal';
+import withWindowDimensions from '../withWindowDimensions';
+
+/*
+ * This is a convenience wrapper around the Modal component for a responsive Popover.
+ * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
+ */
+const Popover = props => (
+    <Modal
+        type={CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
```

Comment:
Hmm I wonder if we can get rid of this rule, I see it in so many places in the codebase at this point.